source/uwp/Renderer/lib/RenderedAdaptiveCard.cpp

```diff
@@ -284,9 +285,21 @@ namespace AdaptiveNamespace
             }
             // else fallthrough
         }
-
         case ABI::AdaptiveCards::Rendering::Uwp::ActionType_OpenUrl:
```

Comment:
>ActionType_OpenUrl [](start = 49, length = 18)

Are we intentionally validating on OpenUrl?

Reply:
my mistake, that was not intended

---
In reply to: [445147141](https://github.com/microsoft/AdaptiveCards/pull/4088#discussion_r445147141) [](ancestors = 445147141)